explorer/src/components/account/NFTokenAccountHeader.tsx

```diff
@@ -0,0 +1,113 @@
+import { NftokenTypes } from "@glow-xyz/nftoken-js";
+import "bootstrap/dist/js/bootstrap.min.js";
```

Comment:
I know you copied this from MetaplexNFTHeader, but we can't possibly start to replicate this pattern. This is 21K, gzipped, of side-effectful, non ESModule JavaScript, just to implement tooltips.

Can you do without the tooltip, or just use a `title` attribute?

Reply:
I don't actually know why Bootstrap was included. I removed it and I can still see the following. So I'll remove it but I can remove it from the `package.json` since it's used in `scss`. 

<img width="406" alt="CleanShot 2022-07-29 at 16 42 42@2x" src="https://user-images.githubusercontent.com/1319079/181840246-39f542b7-9053-4795-85f0-98ce1bd10b51.png">